src/main/java/vazkii/botania/common/item/relic/ItemLokiRing.java

```diff
@@ -62,6 +62,7 @@ public class ItemLokiRing extends ItemRelicBauble implements IExtendedWireframeC
 	private static final String TAG_Y_ORIGIN = "yOrigin";
 	private static final String TAG_Z_ORIGIN = "zOrigin";
 	private static final String TAG_MODE = "mode";
+	private static boolean mode = true;
```

Comment:
This is a singleton - does this work in MP with multiple people with multiple rings and has that been tested?

Reply:
I must have forgotten to remove the boolean. its unneeded. Will fix in a moment.

Reply:
Is this better?  I think this now isn't a singleton right?